content/tt/paka.md

```diff
@@ -21,39 +21,44 @@ PAKA (roughly _The PACK_) was a babyface tag team, active in [Prime Time Wrestli
 
 ### Final feuds and break-up
 
-After the post-Gold Rush reveal that Wiktor Longman was the one who attacked Taras almost two years prior, Longman announced that he would destroy Pawłowski and his allies from PAKA. Since Boro was already seriously injured by him previously, his next target was Taras.
-At [Underground 21](@/e/ptw/2024-04-13-ptw-underground-21.md) Taras lost to and got kayfabe injured by Wiktor Longman to such a degree, that he had to be carried out on a stretcher. It was Taras' final appearance in the PTW ring and he officially left the federation not long after.
-At the same show, [Sinister Kingdom](@/tt/sinister-kingdom.md) orchestrated a celebration of PAKA's demise. They joked around, blew party horns and declared themselves a top team in PTW.
-With Pablo failing to save Taras from Longman in time, he became a last PAKA member standing, intent to get a revenge on Longman. This led to Pablo vs Longman No Holds Barred Match at [Total Blast From The Past](@/e/ptw/2024-05-11-ptw-6.md), which was won by Longman. This marked Pablo's last appearance in PTW, as he [left the federation](@/a/ptw-exits.md) soon after.
-Despite not appearing since September 2023, Boro had not officialy [left the federation](@/a/ptw-exits.md) until June 2024, when he posted his "Last Dance Story" where he clearly supported wrestlers leaving PTW, such as his teammates from PAKA and the members of his ["Green Gang"](@/a/zieloni.md).
+After the post-Gold Rush reveal that Wiktor Longman was the one who attacked Taras almost two years prior, Longman announced that he would destroy Pawłowski and his allies from PAKA. Since Boro had already been seriously injured by him previously, Longman's next target was Taras.
+At [Underground 21](@/e/ptw/2024-04-13-ptw-underground-21.md) Taras lost to Wiktor Longman and was kayfabe injured by him to such a degree that he had to be carried out on a stretcher. It was Taras's final appearance in the PTW ring as he officially left the federation not long after. 
+At the same show, Sinister Kingdom orchestrated a celebration of PAKA's demise. They joked around, blew party horns and declared themselves the top team in PTW.
+
+With Pablo failing to save Taras from Longman in time, he became the last PAKA member standing, intent to get revenge on Longman. This led to Pablo vs Longman No Holds Barred Match at [Total Blast From The Past](@/e/ptw/2024-05-11-ptw-6.md), which was won by Longman. This marked Pablo's last appearance in PTW, as he [left the federation](@/a/ptw-exits.md) soon after.
+
+Despite not appearing since September 2023, Boro had not officialy [left the federation](@/a/ptw-exits.md) until June 2024, when he posted his "Last Dance Story" in which he clearly supported wrestlers leaving PTW, such as his teammates from PAKA and the members of his ["Green Gang"](@/a/zieloni.md).
```

Comment:
"left the federation" twice close to each other, please reword, maybe to "quit the fed", just "quit" or other phrasing.